server/manager/src/main/java/org/apache/accumulo/manager/upgrade/UpgradeCoordinator.java

```diff
@@ -107,10 +110,8 @@ public boolean isParentLevelUpgraded(KeyExtent extent) {
 
   private int currentVersion;
   // map of "current version" -> upgrader to next version.
-  private final Map<Integer,
-      Upgrader> upgraders = Map.of(AccumuloDataVersion.SHORTEN_RFILE_KEYS, new Upgrader8to9(),
-          AccumuloDataVersion.CRYPTO_CHANGES, new Upgrader9to10(),
-          AccumuloDataVersion.ROOT_TABLET_META_CHANGES, new Upgrader10to11());
+  private final Map<Integer,Upgrader> upgraders =
+      Map.of(ROOT_TABLET_META_CHANGES, new Upgrader10to11());
```

Comment:
This might be a problem with our existing upgrader code, but `Map.of` does not guarantee ordering. Taken from [the javadoc for unmodifiable maps](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Map.html#unmodifiable): `The iteration order of mappings is unspecified and is subject to change`

In this case, it's a singleton map... but in previous versions, and in the future, it may not be. We should use a map that preserves the order, because these upgraders need to be executed in order, I believe.

Reply:
Addressed in 7c8ec3ad63.  I'll work a patch for 2.1.1 with only the ordering preserving and hard-coded value removal as a separate PR.